.github/workflows/test.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         with:
           profile: minimal
           toolchain: stable
+          target: aarch64-apple-darwin
```

Comment:
I don't think that component should be installed on non-mac targets. Maybe make a separate step that's only run on mac?